src/Microsoft.ML.CodeGenerator/CodeGenerator/CSharp/CodeGenerator.cs

```diff
@@ -0,0 +1,363 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.ML.AutoML;
+using Microsoft.ML.CodeGenerator.Templates.Console;
+using Microsoft.ML.CodeGenerator.Utilities;
+
+namespace Microsoft.ML.CodeGenerator.CSharp
+{
+    public class CodeGenerator : IProjectGenerator
```

Comment:
Let's make this class `internal`. It is not possible to use this class publicly.